ui/api/src/main/java/org/jboss/forge/addon/ui/input/SelectComponent.java

```diff
@@ -30,4 +31,9 @@
    Converter<VALUETYPE, String> getItemLabelConverter();
 
    IMPLTYPE setItemLabelConverter(Converter<VALUETYPE, String> converter);
+
+   Converter<VALUETYPE, String> getInteractionItemLabelConverter();
+
```

Comment:
Although the idea is interesting, I am not sure we should use another converter here. 
I think the best would be to use the converter returned when UIContext.getProvider().isGUI() returns true.